### DISCUSSION TITLE
how to get lanelet2 map and pointcloud map for carla towns?

### DISCUSSION THREAD

**Discussion by gaokao123** [carla-simulator/carla#5829] [Q&A]
how to get lanelet2 map and pointcloud map for carla towns? is there download address?

**Comment by RaviBeagle** [comment]
You will find OpenDrive Maps of the CARLA Towns under folder:
_CARLA/CarlaUE4/Content/Carla/Maps/OpenDrive_
You would have to convert the xodr to lanelet2 osm files with some tool. 

PCD Maps are found under:
_CARLA/HDMaps_ 

But only for Town01 to Town07

**Reply by gaokao123** [reply]
@RaviBeagle 
First, I tried some tools, but couldn't convert the xodr to lanelet2 osm files. do you have any tools that convert the xodr to lanelet2 osm files successfully?
Second, I found the lanelet2 Maps of the CARLA Towns under folder:
https://bitbucket.org/carla-simulator/autoware-contents/src/master/
but in these maps, there is no traffic light information. Do you have any ideas?  thank you!

**Comment by RaviBeagle** [comment]
https://commonroad.in.tum.de/tools/scenario-designer
I have not tried this. You could give it a shot
